action/store/postgres.py

```diff
@@ -118,5 +118,7 @@ def _clean_column_default(value: str) -> Any:
             value = value[0 : -len("::text")]
             if value[0] == value[-1] == "'" and value.count("'") == 2:
                 value = value[1:-1]
+        if value.isnumeric():
```

Comment:
Do we have any booleans? Would be nice to include them too

Reply:
Yeah, we do. Added.